### ISSUE TITLE
[Issue] - Health Checker

### ISSUE THREAD

**Issue by aStation94** [microsoft/CSS-Exchange#1654]
**Provide Version Number**
23.04.04.1055

**Describe the issue**
The script is telling me that there is a "missing configuration file" and there is an event ID of 1309.  So far OWA and ECP seem okay, but the error worries me.  This happened right after updating cu23 to KB5024296.  Before the update  I was not seeing this issue.  I followed the URL and tried to apply the fix, but it didn't really work because the filepaths mentioned in the url below don't seem to exist in my environment.  
 https://aka.ms/HC-MissingConfig
 
**Expected behavior**
I would expect not to be told I have a missing configuration.  

**Script Output**
CTS Processor Affinity Percentage: 0
        Disable Async Notification: 0
        Credential Guard Enabled: False
        EdgeTransport.exe.config Present: True
        Open Relay Wild Card Domain: Not Set
        Missing Configuration File:
                Missing:
                Missing:
                Missing:
                More Information: https://aka.ms/HC-MissingConfig

**Additional context**
This happened right after updating cu23 to KB5024296.

I have another server in the DAG and it is not reporting this error.

Attached is the event and screenshot.

Thanks for the help! 
![issue](https://user-images.githubusercontent.com/130541645/231381264-9906c719-f8e0-45a9-af04-7edafdfc33d7.PNG)

[event1309.txt](https://github.com/microsoft/CSS-Exchange/files/11208273/event1309.txt)

**Comment by dpaulson45** [comment]
I believe this is related to issue #1618.

**Comment by aStation94** [comment]
> I believe this is related to issue #1618.

I took a look at it and it didn’t make too much sense to me unfortunately.  Could you elaborate or help me understand what’s going on please?

**Comment by aStation94** [comment]
We were able to get this resolved with Alex's help.  Turns out 3 of my virtual libraries' "Physcial Path" in the advanced settings were set to "d:\program file\...." and the actual path it should be pointed to is "d:\program fileS...."

**Comment by dpaulson45** [comment]
Yep, Alex did reach out to me about this as well. I will be working on a fix for this to make it clearer as to what the problem is.
